src/components/ServiceList.jsx

```diff
@@ -80,11 +80,24 @@ const ServiceList = () => {
 
             {hasFilteredResults ? (
               <div>
```

Comment:
i think this can be a React.Fragment, don't believe this div is doing anything.

Reply:
yup fragment is all thats needed.